extensions/amp-analytics/amp-analytics.md

```diff
@@ -624,7 +624,7 @@ indicate which transport methods are acceptable.
 
   - `beacon` Indicates [`navigator.sendBeacon`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon)  can be used to transmit the request. This will send a POST request, with credentials, and an empty body.
   - `xhrpost` Indicates `XMLHttpRequest` can be used to transmit the request. This will send a POST request, with credentials, and an empty body.
-  - `image` Indicates the request can be sent by generating an `Image` tag. This will send a GET request.
+  - `image` Indicates the request can be sent by generating an `Image` tag. This will send a GET request. There is an option to suppress the warnings of empty GET responses by setting "image": {"ignoreWarnings": true}.
```

Comment:
This should be more specific since the warnings can also be caused by real errors. Also use backticks to format code snippets.

"To suppress console warnings due to empty responses or request failures, set `"image": {"suppressWarnings": true}`."